frappe/__init__.py

```diff
@@ -420,10 +420,11 @@ def get_site_config(sites_path: str | None = None, site_path: str | None = None)
 	# Generalized env variable overrides and defaults
 	def db_default_ports(db_type):
 		from frappe.database.mariadb.database import MariaDBDatabase
+		from frappe.database.postgres.database import PostgresDatabase
```

Comment:
Should change this to check db type and accordingly conditionally import, else it breaks for setups without postgres

Reply:
👍 - raised https://github.com/frappe/frappe/pull/28118